tests/integration/targets/filter_hashids/tasks/main.yml

```
@@ -0,0 +1,58 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+- name: Test valid hashable inputs
+  assert:
+    that:
+      - "single_int | community.general.hashids_encode | community.general.hashids_decode | first == single_int"
```

```suggestion
      - "single_int | community.general.hashids_encode | community.general.hashids_decode == [single_int]"
```
Makes sure that the list is actually one element :)